lib/mix/tasks/kitto.install.ex

```diff
@@ -0,0 +1,100 @@
+defmodule Mix.Tasks.Kitto.Install do
+  use Mix.Task
+  @shortdoc "Install community Widget/Job from a Github Gist"
+  @supported_languages ["JavaScript", "SCSS", "Markdown", "Elixir"]
+  @github_url "https://api.github.com/gists/"
+
+  @moduledoc """
+  Installs community Widget/Job from a Github Gist
+
+      mix kitto.install --widget test_widget --gist JanStevens/0209a4a80cee782e5cdbe11a1e9bc393
+      mix kitto.install --gist 0209a4a80cee782e5cdbe11a1e9bc393
```

Comment:
Why is the second evocation missing the owner of the gist. I expected it to also have `JanStevens/0209a4a80cee782e5cdbe11a1e9bc393`